src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -103,6 +104,11 @@ protected function configure(): void
 messages didn't originate from Messenger:
 
     <info>php %command.full_name% <receiver-name> --bus=event_bus</info>
+
+Use the --queues option to limit a receiver to only certain queues. This is only possible for receivers that support restricting
+the queues by implementing the QueueReceiverInterface and leads to an error otherwise:
```

Comment:
Not sure this implementation detail should be part of the description and rather "explain" it by an exception if this happens

Reply:
agreed, shortened the second part to just a parenthesis that not all receivers support this